hs_model_program/models.py

```diff
@@ -149,20 +124,50 @@ def get_xml(self):
         # get the root 'Description' element, which contains all other elements
         container = RDF_ROOT.find('rdf:Description', namespaces=self.NAMESPACES)
 
-        # inject raster resource specific metadata elements into container element
-        fields = ['software_version',
-                  'software_language',
-                  'operating_sys',
-                  'date_released',
-                  'program_website',
-                  'software_repo',
-                  'release_notes',
-                  'user_manual',
-                  'theoretical_manual',
-                  'source_code']
-
-        model_program_object = self.program
-        self.add_metadata_element_to_xml(container, model_program_object, fields)
+        if self.program:
+            model_engine = etree.SubElement(container, '{%s}modelEngine' % self.NAMESPACES['hsterms'])
```

Comment:
@pabitra I added a function to build the xml for multi-element metadata objects.  8f19d273baae438874214ed68c90c535cb27218b